backend/src/Squidex.Domain.Apps.Entities/Contents/Queries/ContentQueryService.cs

```diff
@@ -91,6 +92,13 @@ public async Task<IResultList<IEnrichedContentEntity>> QueryAsync(Context contex
             }
 
             var schema = await GetSchemaOrThrowAsync(context, schemaIdOrName);
+            var permission = Permissions.ForApp(Permissions.AppContentsRead, context.App.Name, schemaIdOrName);
+            var permissionReadOwn = Permissions.ForApp(Permissions.AppContentReadOwn, context.App.Name, schemaIdOrName);
+
+            if (!context.Permissions.Allows(permission) && context.Permissions.Allows(permissionReadOwn))
```

Comment:
It is not necessary to ask for permissionReadOwn here because we know that we have this permissions, otherwise it would fail.